python/src/ExponentialFactory_doc.i.in

```diff
@@ -4,16 +4,30 @@
 Available constructor:
     ExponentialFactory()
 
-The parameters are estimated by method of moments:
+Let :math:`\bar{x}` be the sample mean. 
+Let :math:`x_{(1)}` be the sample minimum. 
+The parameter :math:`\gamma` is estimated by method of maximum 
+likelihood, with asymptotic correction for bias:
 
 .. math::
     :nowrap:
 
     \begin{eqnarray*}
-      \displaystyle \Hat{\gamma}_n = x_{(1,n)} - \frac{|x_{(1,n)}|}{2+n}\\
-      \displaystyle \Hat{\lambda}_n= \frac{1}{\bar{x}_n-\Hat{\gamma}_n}
+      \hat{\gamma} = x_{(1)} - \frac{|x_{(1)}|}{2 + n}
     \end{eqnarray*}
 
+The parameter :math:`\lambda` is estimated by method of moments:
+
+.. math::
+    :nowrap:
+
+    \begin{eqnarray*}
+      \hat{\lambda} = \frac{1}{\bar{x} - \hat{\gamma}}
+    \end{eqnarray*}
+
+When :math:`\gamma` is known, the estimator for :math:`\lambda` is equivalent 
+to the method of maximum likelihood. 
```

Comment:
Is it possible not to estimate \gamma when using `ExponentialFactory`? If not, then I am not sure this sentence is useful here.

Besides, the wording is a little confusing because the reader needs to realize that, in the context of this sentence and this sentence only, \hat{\gamma} is actually \gamma.